virt/virt-2-5-release-notes.adoc

```diff
@@ -112,6 +112,9 @@ The *Pending Changes* banner at the top of the page displays a list of all chang
 
 //CNV-7296 - HCO and HPP CRs moved to v1beta1
 
+//BZ-1874403
+* The default cloud-init user password is now auto-generated for virtual machines that are created from templates.
```

Comment:
Markup "cloud-init" in backticks as a user name (per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).